simple.css

```diff
@@ -252,8 +254,8 @@ header > nav a:visited {
 }
 
 header > nav a:hover {
```

Comment:
Hopefully addressed your comments. Would you like this reverting too? This was included to provide consistency with other hover items like buttons